modules/mavlink/flight_controller.py

```diff
@@ -351,3 +351,16 @@ def insert_waypoint(
         commands.insert(index, new_waypoint)
 
         return self.upload_commands(commands)
+
+    def send_statustext_msg(
+        self,
+        vehicle: dronekit.Vehicle,
+        message: str,
+        severity: int = mavutil.mavlink.MAV_SEVERITY_INFO,
+    ) -> None:
+        """
+        Sends a STATUSTEXT message to the vehicle.
+        """
+        message_bytes = message.encode("utf-8")
```

Comment:
Please check to make sure your message_bytes doesn't go over the capacity limit of 50 bytes. Return a boolean success or not to indicate whether the sending was successful or not